src/components/Pagination/icons.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { ChevronRightLarge, ChevronLeftLarge } from 'wix-ui-icons-common';
+
+export const DoubleChevronLeftLarge = () => (
```

Comment:
@jonathanadler can you help us to get double chevron icon from designers, so we can update it here https://wix-wix-ui-icons-common.surge.sh/?selectedKind=Icons%2FGeneral&selectedStory=Small&full=0&addons=0&stories=1&panelRight=0

Reply:
Sure.
In the meantime you can use the icons from here: https://zeroheight.com/7sjjzhgo2/p/884c1f/t/b83d (click on the icon, and in the top right corner you'll have a knife icon, that will lead you to the asset download button).
I'll check with the designers that this is updated.

But note that our icons are not the same as in wix-ui-icons-common.
We have our own icons folder which we export and also use internally (We now use Yoshi 4, so we can just import SVG's).

Reply:
So I checked with the designers.
The icons I sent you are up to date.
So if you managed to use them, that's perfect.
Just note that you might need to clean them from `id`'s `defs`'s etc.